### PR TITLE
docs: add a section suggesting gifs for pr reviews

### LINKED ISSUE #6889: Docs: Suggest gif reactions on PR maintenance practices
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

@bradzacher I really like how you post gifs on PR reviews - it feels fun and welcoming. Maybe we should standardize this as a suggestion in our PR maintenance docs? I'm imagining it's something we can encourage ourselves to do especially on PR approvals and also on other comments. 

Thanks to @bdougie I now know of [GIFs for GitHub](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep) (and [Firefox support too!](https://addons.mozilla.org/en-US/firefox/addon/gifs-for-github)). Which makes the task of finding a gif much easier.

### Affected URL(s)

https://typescript-eslint.io/maintenance/pull-requests

### PR DESCRIPTION
**Pull request by sfavello**
## PR Checklist

- [x] Addresses an existing open issue: fixes #6889 
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

Added a suggestion section in the pull request docs with a section in adding gifs to PR reviews. I wasn't sure if I should add a new section called suggestions or put it below the first section. I was worried that the first section might become too verbose and that it might be better to break it up this way. I can definitely change/switch it to however is best though, no worries. Just wanted to say that I thought this was a great idea. Adding gifs to PR reviews can really make a difference, imho. 

andy

### CODE REVIEW

File: docs/maintenance/Pull_Requests.mdx [hunk 1]
```diff
@@ -32,3 +32,8 @@ Our flow for PRs that have been waiting for >=1 month is:
 2. Add the `stale` label to the PR
 3. Wait 2 weeks
 4. If they still haven't responded, close the PR with a message like the _Pruning Stale PR_ template
+
```
Comment (sfavello) [inline docs/maintenance/Pull_Requests.mdx:35]:
```suggestion
```

File: docs/maintenance/Pull_Requests.mdx [hunk 2]
```diff
@@ -32,3 +32,7 @@ Our flow for PRs that have been waiting for >=1 month is:
...
+## Suggestions
+
+Add some fun to PR reviews with GIFs. It's a great way to create a fun and welcoming environment while still maintaining a professional and productive one. The benefits of using GIFs in PR reviews are that it adds a touch of humor and lightheartedness to the process, builds camaraderie and team spirit, and can help break the ice by making users feel welcomed. Please remember to use appropriate and respectful GIFs and use them sparingly; GIFs should enhance the conversation, not detract from it. Check out [GIFs for Github here](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep) or [for Firefox here](https://addons.mozilla.org/en-US/firefox/addon/gifs-for-github/). Have some fun with it and happy GIF-ing.
```
Comment (JoshuaKGoldberg) [inline docs/maintenance/Pull_Requests.mdx:38]:
```suggestion
Consider adding fun to PR reviews with GIFs.
The benefits of using GIFs in PR reviews are that it adds a touch of humor and lightheartedness to the process, builds camaraderie and team spirit, and can help break the ice by making users feel welcomed.
Please remember to use appropriate and respectful GIFs and use them sparingly; GIFs should enhance the conversation, not detract from it.

> We like to use the _GIFs for Github_ extension: available as [GIFs for GitHub (Chrome)](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep) and [GIFs for GitHub (Firefox)](https://addons.mozilla.org/en-US/firefox/addon/gifs-for-github). ✨
```

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Sorry this took so long to review @sfavello - it slipped by me. But I really like it! 

Just suggesting a bit of touching up to match the existing style: [one sentence per line](https://github.com/JoshuaKGoldberg/sentences-per-line) and removing a couple duplicated points. What do you think?

Bill and Ted from their movie going 'excellent!' in their rock garage

File: docs/maintenance/Pull_Requests.mdx (hunk 2 above)
Comment (sfavello) [reply docs/maintenance/Pull_Requests.mdx:38]: No worries at all. 😄 
And yes! This looks way more polished. The format makes it so much more easier to read too! 🙌🏼 Thank you!

**Review by JoshuaKGoldberg — APPROVED** [review]
💯 thanks!

**Pull request merged**